### ISSUE TITLE
Does we need to build validation set ?

### ISSUE THREAD

**Issue by doom8199** [2017-fall-DL-training-program/ConvNetwork#6]
Dear TAs:

    I found that 45k/5k train/val split was chosen in the CIFAR-10 example of the original ResNet paper. Does it means that we need to extract 45k data from training set for trainging, and others (5k) for validation in our homework?

    If yes, which validation algorithm need to be implement at least ? (ex: cross validation, or some other greedy method such as provided example code).

    In the example code, test set is used to "improve" the network parameters, but it should be validation set coming from training set

**Comment by JiaRenChang** [comment]
Hi,
The ResNet paper used val split to choose hyper-parameters.
In our homework, we give you whole hyper-parameters, so you don't need validation split.
If you want to tune the hyper-parameters or select the best trained model, you will need val split. 
Jia-Ren

**Comment by doom8199** [comment]
thank you
